bedrock/firefox/templates/firefox/all.html

```diff
@@ -30,7 +30,7 @@
   sky firefox-all-{{ channel }} {% if channel == 'alpha' %}blueprint{% endif %}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
Yeah, seems from a macro level to be an "it depends" situation. Does a cached resource count against the max parallel HTTP requests limit? The home page currently has 30 HTTP requests, and it seems the max parallel limit is around 8 in modern browsers, so maybe it doesn't matter? This is outside of my ken. :smile_cat:

Bring it up in tomorrow's meeting?